components/AuthContainer/AuthContainer.tsx

```diff
@@ -1,30 +1,20 @@
 import { Typography } from "@equinor/eds-core-react"
-import { signIn, useSession } from "next-auth/react"
+import { useSession } from "next-auth/react"
 import type { FunctionComponent, PropsWithChildren } from "react"
 import { useIntl } from "react-intl"
 
-import { Container } from "../Container"
+import { SignIn } from "./SignIn"
 
 export const AuthContainer: FunctionComponent<PropsWithChildren> = ({ children }) => {
   const { data, status } = useSession()
   const intl = useIntl()
 
   if (status === "loading") {
-    return (
-      <Container>
-        <Typography>{intl.formatMessage({ id: "auth.loading" })}</Typography>
-      </Container>
-    )
+    return <Typography>{intl.formatMessage({ id: "auth.loading" })}</Typography>
   }
 
   if (status !== "authenticated" || data?.error === "RefreshAccessTokenError") {
-    signIn("azure-ad")
-
-    return (
-      <Container>
-        <Typography>{intl.formatMessage({ id: "auth.prompt" })}</Typography>
-      </Container>
-    )
+    return <SignIn />
```

Comment:
i think we might want to do this server side and redirect to the route instead of just rendering the view